localisation/00_zuwc_united_localisation.csv

```diff
@@ -2881,13 +2881,13 @@ is_dragon_race_trigger_tooltip;Is a dragon;;;;;;;;;;;;;x
 has_no_class_trait_trigger_tooltip;Has no class;;Hat keine Klasse.;;;;;;;;;;;x
 is_medium_humanoid_tooltip;Is medium humanoid;;;;;;;;;;;;;x
 is_large_humanoid_tooltip;Is large humanoid;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+text_effect_decision_abduct;Enables �GAttempt to Abduct�! on character portraits;;;;;;;;;;;;;x
```

Comment:
Maybe it should be clarified in text effect that vampires can abduct only courtiers?

Reply:
Mentioned in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1274/commits/a002eeccb6ee23b0c75842394c70c88bf7222781.